cse/Ranker.py

```diff
@@ -0,0 +1,61 @@
+import cse.WeightCalculation as wc
```

Comment:
`wc` -> `WeightCalculator` or `WeightCalc` (first letter capital -> no class or object. it's easier to recognize in the code)

Reply:
it's just a file providing some helper functions, so we don't have to change code in the crawling stage itself...should I rename the File also to `weightCalculation` and no alias?

Reply:
Ok I wasn't clear enough. The file name (of WeightCalculation) and it's contents are fine. My problem is `wc`. It looks like an object in the code which is a little bit confusing.